quickevent/app/quickevent/plugins/Receipts/src/receiptswidget.cpp

```diff
@@ -289,18 +289,24 @@ void ReceiptsWidget::markAsPrinted(int connection_id, int card_id)
 void ReceiptsWidget::loadReceptList()
 {
 	ui->lstReceipt->clear();
-	QString receipts_dir = getPlugin<ReceiptsPlugin>()->qmlDir() + "/receipts";
-	QDirIterator it(receipts_dir, QStringList{"*.qml"}, QDir::Files | QDir::Readable, QDirIterator::Subdirectories);
-	while (it.hasNext()) {
-		it.next();
-		QFileInfo fi = it.fileInfo();
-		QString path = fi.filePath();
-		QString name = path.mid(receipts_dir.length() + 1);
-		if(name.startsWith("private/", Qt::CaseInsensitive))
-			continue;
-		name = name.mid(0, name.indexOf(".qml", Qt::CaseInsensitive));
-		qfInfo() << "receipt:" << name << "path:" << path;
-		ui->lstReceipt->addItem(name, path);
+	QVector<QString> receipts_paths = {
+		getPlugin<ReceiptsPlugin>()->qmlDir() + "/receipts",
+		QCoreApplication::applicationDirPath() + "/receipts",
```

Comment:
We have `quickevent-data` directory introduced for custom app style-sheet, custom receipts should be IMO put there as well.
```c++
	QCoreApplication::applicationDirPath() + "/quickevent-data/receipts",
	QDir(".").absolutePath() + "/quickevent-data/receipts"
```

Reply:
ok.
offtopic: any idea why is the `sound` folder inside `style` folder?

Reply:
because sounds can be customized as well, it is not visual style only

Reply:
but why this folder structure, sounds are not related to style(sheets),  just `/quickevent-data/sounds` seems more logical to me
resulting in
`/quickevent-data/styles`
`/quickevent-data/sounds`
`/quickevent-data/receipts`

Reply:
well, `/quickevent-data/sound` sounds better, please create MR if you think that it worth to do it.